FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -13,12 +13,42 @@ public final class RemoteFeedLoader: FeedLoader {
 		case invalidData
 	}
 
+	public struct Root: Decodable {
+		let items: [Item]
+	}
+
 	public init(url: URL, client: HTTPClient) {
 		self.url = url
 		self.client = client
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			
+			guard self != nil else { return }
+			
+			switch result {
+				case let .success((data, response)):
+					if response.statusCode == 200, let images = try? JSONDecoder().decode(Root.self, from: data) {
+						completion(.success(images.items.map({ $0.item })))
+					} else {
+						completion(.failure(Error.invalidData))
+					}
+				case .failure:
+					completion(.failure(Error.connectivity))
+			}
+			return
+		}
+	}
+}
+
+internal struct Item: Decodable {
+	let image_id: UUID
+	let image_desc: String?
+	let image_loc: String?
+	let image_url: URL
+
+	var item: FeedImage {
```

Comment:
It is better to call this property `image` to better describe the returning type and not to confuse it with `Item`

```suggestion
	var image: FeedImage {
```